### DISCUSSION TITLE
What is the pnpm equivalent of `npm:findConflictingDependencies` or `yarn:findConflictingDependencies` if any?

### DISCUSSION THREAD

**Discussion by yeikel** [pnpm/pnpm#7071] [Q&A] [answered]
As the title states, what is the pnpm equivalent of `npm:findConflictingDependencies` or `yarn:findConflictingDependencies` if any?

**Comment by zkochan** [comment, marked as answer]
I have no idea what findConflictingDependencies is. Never heard about it.

**Comment by yeikel** [comment]
Sorry about the confusion. it turns out that this was just a script in a build and not an actual npm step. Closing and thanks for the input
